Example/Example/SeriesDetailCollectionViewDataSource.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  SeriesDetailCollectionViewDataSource.swift
+//  Example
+//
+//  Created by Pedro Vicente Gomez on 23/12/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BothamUI
+
+class SeriesDetailCollectionViewDataSource<U, V: BothamViewCell where U == V.ItemType> : BothamCollectionViewDataSource<U, V> {
```

Comment:
Fixed!!! Are you reading my mind?
![Hermione](https://media2.giphy.com/media/CZgYXF0F9FZPa/200_s.gif)